server.js

```diff
@@ -62,30 +57,33 @@ app.post('/polls', function(request, response) {
 app.get('/polls/:id', function(request, response) {
   var id = request.params.id;
 
+  // TODO use id to actually find what i'm looking for :)
+
   redis.hgetall("polls", function (err, poll) {
 
     var host = request.protocol + '://' + request.get('host') + "/polls/"
 
-    poll.adminUrl = host + poll.adminString
-    poll.voterUrl = host + poll.voterString
+    formatData(poll, host);
 
-
-    poll.choices = poll.choices.split(',')
     response.render('poll', { poll: poll })
 
   });
 
 });
 
+function formatData(poll, host) {
```

Comment:
Kinda. I want to pull out Poll eventually, but after talking to Meeka I am going to work on tying the voting together to start tallying votes